packages/parameters/tests/helpers/cdkAspectGrantAccess.ts

```diff
@@ -30,12 +35,27 @@ export class ResourceAccessGranter implements IAspect {
     if (node instanceof NodejsFunction) {
 
       // Grant access to the resources
-      this.resources.forEach((resource: Table | Secret) => {
+      this.resources.forEach((resource: Table | Secret | StringParameter | IStringParameter) => {
 
         if (resource instanceof Table) {
           resource.grantReadData(node);
-        } else if (resource instanceof Secret) {
+        } else if (
+          resource instanceof Secret
+        ) {
+          resource.grantRead(node);
+        } else if (isStringParameterGeneric(resource)) {          
           resource.grantRead(node);
+          // Grant access also to the path of the parameter
+          node.addToRolePolicy(
+            new PolicyStatement({
+              actions: [
```

Comment:
Is `Effect.ALLOW` not necessary, or implicit, or forgot to add?

Reply:
It should be implicit, since it runs successfully & it didn't before me adding this statement.

However I agree it could be made even more explicit so I'll add it.

Reply:
TIL, but adding explicit helps.